sys-devel/gettext/gettext-0.19.7.recipe

```diff
@@ -97,7 +107,6 @@ INSTALL()
 	make install AUTOCONF=: AUTOMAKE=: AUTOHEADER=: ACLOCAL=:
 
 	rm $libDir/charset.alias
-	rm $libDir/lib*.la
```

Comment:
Have a look in https://github.com/haikuports/haikuports/blob/master/media-libs/freetype/freetype-2.6.3.recipe.